pkg/apis/core/validation/validation.go

```diff
@@ -911,9 +911,13 @@ func validateQuobyteVolumeSource(quobyte *core.QuobyteVolumeSource, fldPath *fie
 			}
 		}
 	}
-
+	quobyteVolFmt := regexp.MustCompile(`[^\w\s/-]`)
```

Comment:
nit: I think `strings.ContainsAny` is probably the more efficient approach.

Reply:
Yes, `strings.ContainsAny` is more efficient but doesn't fit the current validation scenario as Quobyte volume name only supports ASCII Characters and only subset of special characters . For example, Quobyte UI doesn't support unicode characters, with `strings.ContainsAny` it is cumbersome to validate the volume name.

This validation is only for creation of resource, so IMO it is not a big concern. I will move the `regexp.MustCompile(`[^\w\s/-]`)` to global variable which makes it little more efficient.